package.json

```diff
@@ -72,7 +72,7 @@
     "test": "pnpm run --recursive test",
```

Comment:
there are a lot of scripts here with a test prefix, I'm a bit at a loss to which one should be tasked with starting the unit tests.